apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -847,6 +847,46 @@ describe('UserService', () => {
 		});
 	});
 
+	describe('deleteUser', () => {
+		describe('when user is successfully deleted', () => {
+			const setup = () => {
+				const userId = new ObjectId().toHexString();
+
+				userRepo.deleteUser.mockResolvedValueOnce(1);
+
+				return { userId };
+			};
+
+			it('should return true', async () => {
+				const { userId } = setup();
+
+				const result = await service.deleteUser(userId);
+
+				expect(result).toBe(true);
+				expect(userRepo.deleteUser).toHaveBeenCalledWith(userId);
+			});
+		});
+
+		describe('when user deletion fails', () => {
```

Comment:
I think "fails" is not really correct. If it fails there would be an exception. The user might also just not exist
```suggestion
		describe(`when user was'nt deleted`, () => {
```

Reply:
Done